pulp_ansible/app/models.py

```diff
@@ -65,14 +65,30 @@ class CollectionVersion(Content):
 
     TYPE = "collection_version"
 
+    authors = psql_fields.JSONField(default=list, editable=False)
+    contents = psql_fields.JSONField(default=list, editable=False)
+    description = models.TextField(blank=True, editable=False)
+    documentation = psql_fields.JSONField(default=dict, editable=False)
+    license = psql_fields.JSONField(default=list, editable=False)  # list?
+    name = models.CharField(max_length=32, editable=False)
+    namespace = models.CharField(max_length=32, editable=False)
+    quality_score = models.FloatField(null=True, editable=False)
+    readme = models.TextField(blank=True, editable=False)  # Is this really the filename?
```

Comment:
Since the importer will return all that is in the manifest, it will likely keep returning this value (the filename), but as discussed earlier the readme text (and many other documentation bits) will be returned in docs json blob - so perhaps not model this?

Reply:
I agree with you. Is it in the spec though? If it's in there and we remove it we won't be compatible. I'm not sure if it is or isn't.

Reply:
It's in the spec so the collection author must include it, and the importer will use it to find the readme file and process it, but as far as post-import storing it in the database or not... i dont think that changes compatibility

Reply:
I left it in because the whole model adheres to the spec right now.